src/core/Akka/Dispatch/AbstractDispatcher.cs

```diff
@@ -116,6 +116,26 @@ protected ExecutorServiceConfigurator(Config config, IDispatcherPrerequisites pr
         public IDispatcherPrerequisites Prerequisites { get; private set; }
     }
 
+    internal sealed class ChannelExecutorConfigurator : ExecutorServiceConfigurator
```

Comment:
Used to configure the `TaskSchedulerExecutor(id, new FixedConcurrencyTaskScheduler(MaxParallelism));`